\begin{longtable}{@{}p{3.5cm}@{}p{\dimexpr\textwidth-1cm\relax}@{}}
\nomen{ACN}{acetonitrile}
\nomen{ATP} {adenosine triphosphate}
\nomen{BLAST}{Basic local alignment search tool}
\nomen{C\#}{C sharp, a programming language}
\nomen{CP}{Core Protease}
\nomen{Da}{Dalton, the atomic mass unit}
\nomen{DMSO}{Dimethyl sulfoxide}
\nomen{dNSAF}{distributed Normalized Spectral Abundance Factor}
\nomen{DTT}{Dithiothreitol}
\nomen{DUB}{De-ubiquitylating enzyme}
\nomen{E1}{Ubiquitin Activating Enzyme}
\nomen{E2}{Ubiquitin Conjugating Enzyme}
\nomen{E3}{Ubiquitin Ligase}
\nomen{ESI}{Electrospray ionization}
\nomen{E-value}{Expectation value}
\nomen{FA}{formic acid}
\nomen{FASTA}{A format for storing protein sequences}
\nomen{FDR}{False discovery rate}
\nomen{GB}{Gigabytes}
\nomen{GUI}{Graphical user interface}
\nomen{h}{Hour}
\nomen{HbYX}{Hydrophoic amino acid, followed by a Tyrosine, followed by any amino acid}
\nomen{HCD}{Higher-energy collisional dissociation}
\nomen{HEPES}{4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid}
\nomen{HPLC}{High-performance liquid chromatography}
\nomen{JA}{Jasmonic acid}
\nomen{kDa}{Kilodalton, the atomic mass unit}
\nomen{KS}{Kolmogorov-Smirnov}
\nomen{LC}{Liquid chromatography}
\nomen{LFQ}{Label-free quantification}
\nomen{\textit{m/z}}{mass to charge ratio}
\nomen{MES}{2-(N-morpholino)ethanesulfonic acid}
\nomen{min}{Minute}
\nomen{MS/MS}{Tandem mass spectrometry}
\nomen{MS}{Mass spectrometry}
\nomen{MS1}{Survey mass analysis}
\nomen{MS2} {Fragment mass spectrum}
\nomen{MSpC}{Morpheus Spectral Counter}
\nomen{NCE}{Normalized collision energy}
\nomen{nLC}{Nanoflow liquid chromatography}
\nomen{NSAF}{Normalized Spectral Abundance Factor}
\nomen{PAGE}{Polyacrylamide gel electrophoresis}
\nomen{PCR}{Polymerase chain reaction}
\nomen{PDB}{Protein Data Bank}
\nomen{PGPH}{Peptidylglutamyl-peptide hydrolyzing}
\nomen{pK$_{a}$}{Acid dissociation constant}
\nomen{PMSF}{Phenylmethane sulfonyl fluoride}
\nomen{ppm}{Part per million}
\nomen{PRIDE}{PRoteomics IDEntifications database}
\nomen{PSI-BLAST}{Position-specific iterated basic local alignment search tool}
\nomen{PSM}{Peptide-spectrum match}
\nomen{PTM}{Post-translational modification}
\nomen{.raw}{Thermo Raw File}
\nomen{RP}{Regulatory Particle}
\nomen{RT-PCR}{Reverse transcription polymerase chain reaction}
\nomen{S/N}{Signal-to-noise ratio}
\nomen{s}{Second}
\nomen{SDS}{Sodium dodecyl sulfate}
\nomen{SILAC}{Stable isotope labeling by amino acids in cell culture}
\nomen{SILAC}{Stable isotope labeling by amino acids in cell culture}
\nomen{TAIR10}{The Arabidopsis Information Resource v10}
\nomen{TEMED}{N,N,N$^{\prime}$,N$^{\prime}$-tetramethylethylenediamine}
\nomen{TFA}{trifluoroacetic acid}
\nomen{TMT}{Tandem mass tag}
\nomen{TPP}{Trans-proteomic pipeline}
\nomen{UBA}{ubiquitin associated}
\nomen{UBL}{ubiquitin like}
\nomen{uNSAF}{unique Normalized Spectral Abundance Factor}
\nomen{UPLC}{ultra-high performance liquid chromatography}
\nomen{UPS}{Ubiquitin 26S Proteasome System}
\nomen{UPS2}{Universal Proteomics Standard 2}
\nomen{UTR}{Untranslated region}
\nomen{Y2H}{Yeast-two-hybrd}
\nomen{YFP}{Yellow fluorescent protein}
\end{longtable}